Bookish/Bookish.Web/Controllers/HomeController.cs

```diff
@@ -25,10 +25,10 @@ public IActionResult Index()
             return View();
         }
 
-        public IActionResult Catalogue(string searchTerm = "")
+        public IActionResult Catalogue(string searchTerm = "", int pageNumber = 1)
```

Comment:
What happens if the user changes the url to have a page number greater than the number of pages? Or changes it to 0 or a negative number? 🙂